lib/pbio/drv/usb/stm32_usbd/usbd_desc.c

```diff
@@ -90,6 +106,126 @@ uint8_t USBD_DeviceDesc[USB_LEN_DEV_DESC] __ALIGN_END = {
 
+/** BOS descriptor. */
+__ALIGN_BEGIN static uint8_t USBD_BOSDesc[USB_SIZ_BOS_DESC] __ALIGN_END =
+{
+    5,                        /* bLength */
+    USB_DESC_TYPE_BOS,        /* bDescriptorType = BOS */
+    LOBYTE(USB_SIZ_BOS_DESC), /* wTotalLength */
+    HIBYTE(USB_SIZ_BOS_DESC), /* wTotalLength */
+    0x01,                     /* bNumDeviceCaps = 1 */
+
+    28,                       /* bLength */
+    USB_DEVICE_CAPABITY_TYPE, /* bDescriptorType = Device Capability */
+    0x05,                     /* bDevCapabilityType = Platform */
+    0x00,                     /* bReserved */
+
+    /*
+     * PlatformCapabilityUUID
+     * Microsoft OS 2.0 descriptor platform capability ID
+     * D8DD60DF-4589-4CC7-9CD2-659D9E648A9F
+     * RFC 4122 explains the correct byte ordering
+     */
+    0xDF, 0x60, 0xDD, 0xD8,           /* 32-bit value */
+    0x89, 0x45,                       /* 16-bit value */
+    0xC7, 0x4C,                       /* 16-bit value */
+    0x9C, 0xD2,
+    0x65, 0x9D, 0x9E, 0x64, 0x8A, 0x9F,
+
+    0x00, 0x00, 0x03, 0x06,           /* dwWindowsVersion = 0x06030000 for Windows 8.1 Build */
+    LOBYTE(USBD_SIZ_MS_OS_DSCRPTR_SET), /* wMSOSDescriptorSetTotalLength */
+    HIBYTE(USBD_SIZ_MS_OS_DSCRPTR_SET), /* wMSOSDescriptorSetTotalLength */
+    USBD_MS_VENDOR_CODE,              /* bMS_VendorCode */
+    0x00                              /* bAltEnumCode = Does not support alternate enumeration */
+};
+
+__ALIGN_BEGIN const uint8_t USBD_OSDescSet[USBD_SIZ_MS_OS_DSCRPTR_SET] __ALIGN_END =
+{
+    0x0A, 0x00,                       /* wLength = 10 */
+    0x00, 0x00,                       /* wDescriptorType = MS_OS_20_SET_HEADER_DESCRIPTOR */
+    0x00, 0x00, 0x03, 0x06,           /* dwWindowsVersion = 0x06030000 for Windows 8.1 Build */
+    LOBYTE(USBD_SIZ_MS_OS_DSCRPTR_SET), /* wTotalLength */
+    HIBYTE(USBD_SIZ_MS_OS_DSCRPTR_SET), /* wTotalLength (cont.) */
+
+    0x14, 0x00,                       /* wLength = 20 */
+    0x03, 0x00,                       /* wDescriptorType = MS_OS_20_FEATURE_COMPATBLE_ID */
+    'W', 'I', 'N', 'U', 'S', 'B',     /* CompatibleID */
+    0x00, 0x00,                       /* CompatibleID (cont.) */
+    0x00, 0x00, 0x00, 0x00,           /* SubCompatibleID */
+    0x00, 0x00, 0x00, 0x00,           /* SubCompatibleID (cont.) */
+
+    0x82, 0x00,                       /* wLength = 130 */
+    0x04, 0x00,                       /* wDescriptorType = MS_OS_20_FEATURE_REG_PROPERTY */
+    0x07, 0x00,                       /* wStringType = REG_MULTI_SZ */
+    /* wPropertyNameLength = 42 */
+    0x2A, 0x00,
+    /* PropertyName = DeviceInterfaceGUIDs */
+    'D', '\0',
+    'e', '\0',
+    'v', '\0',
+    'i', '\0',
+    'c', '\0',
+    'e', '\0',
+    'I', '\0',
+    'n', '\0',
+    't', '\0',
+    'e', '\0',
+    'r', '\0',
+    'f', '\0',
+    'a', '\0',
+    'c', '\0',
+    'e', '\0',
+    'G', '\0',
+    'U', '\0',
+    'I', '\0',
+    'D', '\0',
+    's', '\0',
+    '\0', '\0',
```

Comment:
```suggestion
    0x01, 0x00,                       /* wStringType = REG_SZ */
    /* wPropertyNameLength = 40 */
    0x28, 0x00,
    /* PropertyName = DeviceInterfaceGUID */
    'D', '\0',
    'e', '\0',
    'v', '\0',
    'i', '\0',
    'c', '\0',
    'e', '\0',
    'I', '\0',
    'n', '\0',
    't', '\0',
    'e', '\0',
    'r', '\0',
    'f', '\0',
    'a', '\0',
    'c', '\0',
    'e', '\0',
    'G', '\0',
    'U', '\0',
    'I', '\0',
    'D', '\0',
    '\0', '\0',
```

The device can't be seen with WinRT APIs on Windows unless we change this. (also need to update `USBD_SIZ_MS_OS_DSCRPTR_SET` to match)